src/main/java/racing/util/RandomGenerator.java

```diff
@@ -0,0 +1,10 @@
+package racing.util;
+
+import java.util.Random;
+
+public class RandomGenerator {
+    static public int generate() {
```

Comment:
자바는 말이 많은 언어라 static, public, final... 뭐가 많이 붙어요
그래서 우테코를 포함해서 많은 개발자들이 흔히 사용하는 google style convention 이 있는데요,
[거기서 이 키워드의 순서를 정하고 그대로 사용하자는 약속을 하고 있어요
](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)

그걸 적용해본다면 다음과 같이 바뀌겠네요!
```suggestion
    public static int generate() {
```
컨벤션을 한 번 숙지해보시고, 앞으로 이렇게 작성해보길 권장해요